geps/gep-4152/index.md

```diff
@@ -0,0 +1,63 @@
+# GEP-4152: Extending TLS Validation in BackendTLSPolicy
+
+* Issue: [#4152](https://github.com/kubernetes-sigs/gateway-api/issues/4152)
+* Status: Provisional
+
+## TLDR
+
+The ability for the `BackendTLSPolicy` to skip TLS verification or to validate
+certificates based on their fingerprint or public key hash.
+
+## Motivation
+
+The current `BackendTLSPolicy` follows a secure-by-default approach that requires
+users to provide a trusted CA certificate bundle or rely on the system’s default
+certificate store (which typically includes root CAs) to validate backend server
+certificates. However, real-world deployments include cases where strict
+certificate validation may not be possible or practical, e.g., Development and
+testing environments that use self-signed certificates generated dynamically at
+runtime.
+
+In such scenarios, users may need the flexibility to disable certificate
+verification or to use certificate pinning. Certificate pinning offers a safer
+and more controlled alternative, instead of bypassing TLS validation, the gateway
+verifies that the backend’s certificate matches a known fingerprint or public key  extendedBy:head of managing full certificate chains or trusted
```

Comment:
`extendedBy:head` 

Is this something that was missing to be removed?

Reply:
🙈